app/helpers/users_helper.rb

```diff
@@ -0,0 +1,8 @@
+module UsersHelper
+  # Returns the Gravatar for the given user.
+  def gravatar_for(user)
```

Comment:
this most likely should be a model concern, not an application helper

Reply:
i will use a gem for this